bin/check-snmp.rb

```diff
@@ -31,7 +31,8 @@ class CheckSNMP < Sensu::Plugin::Check::CLI
          default: '127.0.0.1'
 
   option :port,
-         short: '-p port',
+         short: '-P port',
```

Comment:
I saw your note on the changelog but I see no prefix option. Can you point me to it?

Reply:
`metrics-snmp` and `metrics-snmp-bulk` have `-p | --prefix` flags.